.gitattributes

```diff
@@ -0,0 +1 @@
+*.go text eol=lf
```

Comment:
Why `lf` only for `*.go` files? I would like to have `lf` for all text files (especially but not limited to: `*.peg`, `*.md`).

Reply:
yes. i miss `*.peg`
because git sometime can change endline in binaries files  when used `* text eol=lf` (it is specified in manual). autodetect can work incorrectly.
lf endlines important only in `static_code.go` , `static_code_range_table.go` , because  it is directly read by bootstrap parser, and in `*.peg` files, because parser save position for them in grammar.
Other text files autoconverted from crlf to lf  after commit with windows git  standart settings .